### ISSUE TITLE
[Bug] New member sign up notification email address is still noreply@domain.com instead of support email or config smtp email

### ISSUE THREAD

**Issue by MFYDev** [TryGhost/Ghost#21423]
### Issue Summary

Set both email address in portal settings and smtp config as hi@mydomain.com, the sign up user received the email from the correct email address which is hi@mydomain.com, however the website owner's notification of new sign up still from noreply@domain.com which is not even anywhere in the settings

Found tons of related issues and all of them got closed or the developer said it is fixed, however this is still happening, and should be very easy to reproduce.

The funny part is I **reported this on Ghost Community 6 months ago (you can imagine how frustrated I am now because this bug is still here)**, and the contributor found that and even replied to somebody else here: https://github.com/TryGhost/Ghost/issues/17922#issuecomment-1705704808

https://forum.ghost.org/t/staff-new-sign-ups-email-notifications-has-a-wrong-from-email-address/40671

**However I do not know why this issue got closed. Half a year passed, this issue is still happening**

![image](https://github.com/user-attachments/assets/72903bcf-c3e7-4c1b-a616-6cd282cf6663)

related: 
- https://github.com/TryGhost/Ghost/pull/19617
- https://github.com/TryGhost/Ghost/issues/20381
- https://github.com/TryGhost/Ghost/issues/12802
- https://github.com/TryGhost/Ghost/issues/17922
- https://github.com/TryGhost/Ghost/issues/17244

**This is not fixed**

Til this point I can safely assume every notification email send to the staff or owner no matter new user sign up or anything else are still from noreply@domain.com

### Steps to Reproduce

1. set hi@domain.com in config
2. set hi@domain.com in portal settings as support email
3. use test email sign up
4. Website owner should receive new sign up notification from hi@domain.com, but it is from noreply@domain.com

### Ghost Version

5.98.0

### Node.js Version

20

### How did you install Ghost?

selfhosting on ubuntu

### Database type

MySQL 8

### Browser & OS version

Edge latest 

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by MFYDev** [comment]
![image](https://github.com/user-attachments/assets/28604942-d826-4055-aa9d-09389b718d87)

**Comment by MFYDev** [comment]
Til this point I can safely assume every notification email send to the staff or owner no matter new user sign up or anything else are still from noreply@domain.com

**Comment by MFYDev** [comment]
The funny part is I reported this on Ghost Community 6 months ago, and the contributor found that and even reply to somebody else here: https://github.com/TryGhost/Ghost/issues/17922#issuecomment-1705704808

https://forum.ghost.org/t/staff-new-sign-ups-email-notifications-has-a-wrong-from-email-address/40671

However I do not know why this issue got closed. Half a year passed, this issue is still happening

**Comment by kevinansfield** [comment]
I've re-opened [the original issue](https://github.com/TryGhost/Ghost/issues/21423) to avoid duplicates.

**Comment by markstos** [comment]
@kevinansfield Rather than linking back to a different issue, you accidentally make a circular reference back to this same ticket. Looks like you meant to link to:

https://github.com/TryGhost/Ghost/issues/17922

**Comment by vsviridov** [comment]
Ghost 6 seems to ignore `mail__from` when passed via env vars. new sign-ups go via `no-reply@domain.com`

**Comment by itsjakobx** [comment]
Yes, especially during sign up this issue happens:

In my case i don't have ghost.\<mydomain> setup but I have setup \<mydomain> via mail__from on resend.

I got the following error message that might help to pinpoint, the exact location where the code might be slightly wrong.

This "helping" error happens when the user clicks: Sign in -> Sign up -> (Enter Data) -> Sign up 

**Browser Console Error**
<img width="2378" height="726" alt="Image" src="https://github.com/user-attachments/assets/2a55c1f2-3444-4539-ad71-d98d8b0965b6" />

Note: I redacted my domain by replacing it with example.com.

**Logs**
```
ghost-1                | Failed to send email. Reason: Message failed: 450 The ghost.example.com domain is not verified. Please, add and verify your domain on https://resend.com/domains.
ghost-1                |
ghost-1                | "Please see https://ghost.org/docs/config/#mail for instructions on configuring email."
ghost-1                |
ghost-1                | Error ID:
ghost-1                |     2e74b080-e8fb-11f0-80f8-49f03d95fd36
ghost-1                |
ghost-1                | Error Code:
ghost-1                |     EMESSAGE
ghost-1                |
ghost-1                | ----------------------------------------
ghost-1                |
ghost-1                | Error: Message failed: 450 The ghost.example.com domain is not verified. Please, add and verify your domain on https://resend.com/domains
ghost-1                |     at createMailError (/var/lib/ghost/versions/6.10.3/core/server/services/mail/GhostMailer.js:85:12)
ghost-1                |     at SMTPConnection._formatError (/var/lib/ghost/versions/6.10.3/node_modules/nodemailer/lib/smtp-connection/index.js:809:19)
ghost-1                |     at SMTPConnection._actionSMTPStream (/var/lib/ghost/versions/6.10.3/node_modules/nodemailer/lib/smtp-connection/index.js:1730:34)
ghost-1                |     at SMTPConnection.<anonymous> (/var/lib/ghost/versions/6.10.3/node_modules/nodemailer/lib/smtp-connection/index.js:1198:22)
ghost-1                |     at SMTPConnection._processResponse (/var/lib/ghost/versions/6.10.3/node_modules/nodemailer/lib/smtp-connection/index.js:993:20)
ghost-1                |     at SMTPConnection._onData (/var/lib/ghost/versions/6.10.3/node_modules/nodemailer/lib/smtp-connection/index.js:774:14)
ghost-1                |     at SMTPConnection._onSocketData (/var/lib/ghost/versions/6.10.3/node_modules/nodemailer/lib/smtp-connection/index.js:195:44)
ghost-1                |     at TLSSocket.emit (node:events:519:28)
ghost-1                |     at addChunk (node:internal/streams/readable:561:12)
ghost-1                |     at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
ghost-1                |     at Readable.push (node:internal/streams/readable:392:5)
ghost-1                |     at TLSWrap.onStreamRead (node:internal/stream_base_commons:189:23)
```

Configuration next to my docker-compose at .env

(Redacted with example.com)
```
mail__transport=SMTP
mail__options__host=smtp.resend.com
mail__options__port=2465
mail__options__secure=true
mail__options__auth__user=\<redacted>
mail__options__auth__pass=\<redacted>
mail__from="'Noreply' <noreply-ghost@example.com>"
```

**Comment by 9larsons** [comment]
Catching this up, because the situation has changed since it was filed.

**The original report is resolved on current Ghost.** Staff/new-signup notifications going out from `noreply@` instead of your configured address was fixed by the sender-alignment work in #19094 (Nov 2023, ~a year before this issue). Staff notifications now build their from-address from `mail.from`, falling back to `noreply@<domain>` only when `mail.from` is unset. With `mail.from` configured, they use it.

That points to the env var not actually being present in the Ghost process. I'd ask to confirm, but this has been open quite a while, so I'm going to close this out. Please re-open with a new issue if you encounter this still.
